src/services/contact/contact.ts

```diff
@@ -30,7 +31,7 @@ export default function contact(req: Request, res: Response, next: NextFunction)
   };
 
   db['contact'].create(newContact)
-    .then(user => contactEmail(newContact)
+    .then(user => contactEmail({ ...newContact, code })
       .then(() => res.status(200).json({ msg: 'Contact request saved' }))
       .catch((err) => {
         log(`Unexpected error ${err}`);
```

Comment:
let's log error text messages, specific and descriptive to each expected error, e.g. `connection to the smtp service failed, email not set` and `contact not saved`.

consider that `caught` is just another block, and can be used between other blocks, e.g. `db['contact'].create(newContact).then().catch().then()...`

please ensure that all failed or aborted actions are reported. In a case where an operation consists of 5 steps, and the fail happened at step 3., then the logs should display the errors for step 3. and also steps 4. and 5.